<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PlotMainWindow</class>
 <widget class="QMainWindow" name="PlotMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>649</width>
    <height>542</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="plotHorizontalLayout">
      <item>
       <layout class="QVBoxLayout" name="plotVerticalLayout" stretch="0,0,0,0,0,0,0,0,0,0,0,0">
        <item>
         <widget class="QComboBox" name="plotComboBox">
          <property name="maximumSize">
           <size>
            <width>128</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="currentIndex">
           <number>0</number>
          </property>
          <item>
           <property name="text">
            <string>Area</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>CurvTot</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>CurvBulk</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>CurvHull</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>PolTotalLength</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>PolTotalComps</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>PolBulkLength</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>PolBulkComps</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>PolHullLength</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>PolHullComps</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Boid Angle</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>HullAngHist</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>HullCompsHist</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>HulldAngHist</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>HullRadHist</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="boidSelectorSpinBox">
          <property name="prefix">
           <string>Boid:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="plotPlotAllPushButton">
          <property name="text">
           <string>Plot All</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="plotLineEdit"/>
        </item>
        <item>
         <widget class="QPushButton" name="printPushButton">
          <property name="maximumSize">
           <size>
            <width>128</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>Print to file</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="plotMinSpinBox"/>
        </item>
        <item>
         <layout class="QHBoxLayout" name="plotMaxHorizontalLayout">
          <item>
           <widget class="QSpinBox" name="plotMaxSpinBox">
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>99</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="plotMaxPushButton">
            <property name="maximumSize">
             <size>
              <width>16</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>+</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QPushButton" name="plotPushButton">
          <property name="text">
           <string>Plot!</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="plotLogxCheckBox">
          <property name="text">
           <string>Log X</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="plotLogyCheckBox">
          <property name="text">
           <string>Log Y</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="updateCheckBox">
          <property name="text">
           <string>Update</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="plotVerticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QCustomPlot" name="plotWidget" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
          <horstretch>5</horstretch>
          <verstretch>5</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>500</width>
          <height>500</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
